src/OrchardCore.Modules/OrchardCore.Media/Assets/scss/_variables.scss

```diff
@@ -1,26 +1,6 @@
-// duplicates of bootstrap.
-$gray-base: #000 !default;
-$gray-dark: lighten($gray-base, 20%) !default; // #333
-$gray: lighten($gray-base, 33.5%) !default; // #555
- 
-$gray-100: #f8f9fa !default;
-$gray-200: #e9ecef !default;
-$gray-300: #dee2e6 !default;
-$gray-400: #ced4da !default;
-$gray-500: #adb5bd !default;
-$gray-600: #6c757d !default;
-$gray-700: #495057 !default;
-$gray-800: #343a40 !default;
-$gray-900: #212529 !default;
-
-$blue: #007bff !default;
-
-$primary: $blue;
-$secondary: $gray-600;
-
-$zindex-fixed: 1030 !default;
+@import "../../node_modules/bootstrap/scss/functions";
+@import "../../node_modules/bootstrap/scss/variables";
```

Comment:
The only risk I can see with this is that if we modify the bootstrap variables in the admin there, the modifications will not get used here. This assumes we will always use the stock bootstrap variables.

Reply:
No it's just using the bootstrap variable declaration file. To get them in the Media gallery style. See what it transpile into. We don't add any vars just use them. In fact we don't want to modify them just use them for consistency.

Reply:
TheAdmin theme uses the same technique of importing the Bootstrap variables. When you just want to use the function, vars and mixins you can just import those without importing all the bootstrap classes. So, if you want to customize TheAdmin theme you can change colors by overriding vars just like they do in Bootswatch.

Reply:
The only way we could affect these variables from the TheAdmin theme would be to add a _variable file import from the TheAdmin theme in here. I'm working on TheAdminDark theme on a different PR maybe we could do that. But it would mean that this module would be now dependent on TheAdmin theme which probably should not.

We need to be able to generate these styles per admin theme if that's what you are suggesting and right now we could'nt and we still can't

Reply:
Food for thoughts on a different PR.

Reply:
@Skrypt I think there is a way to make a custom theme as a separate project / folder. We could take parts of what is in TheAdmin and make it available to any other projects that references it. This way, our admin interfaces from all the different modules could easily reference the admin varialbes and other functions.

Reply:
Yeah let's move this discussion over to : https://github.com/OrchardCMS/OrchardCore/pull/7326